### PR TITLE
[bug 1085618] Use bulk_create in badge tests.

### PR DESCRIPTION
**Pull request by rlr**
I was looking at test performance for like an hour last week. I didnt find anything groundbreaking but... This is a decent speedup for only 4 tests. So it's only like 5 seconds, not much in the grand scheme of things but I figured it doesnt hurt. Might add up to that hour I spent over the next year or decade.

r?

### CODE REVIEW

File: kitsune/customercare/tests/test_badges.py [hunk 1]
```diff
@@ -21,8 +22,12 @@ def test_aoa_badge(self):
...
 
         # Create 49 replies.
-        for i in range(49):
-            reply(user=u, save=True)
+        replies = []
+        for i in range(48):
+            replies.append(reply(user=u))
+        Reply.objects.bulk_create(replies)
+
+        reply(user=u, save=True)
```
Comment (mythmon) [inline kitsune/customercare/tests/test_badges.py]: You should add the comment about signals to this test too.

**Comment by mythmon** [comment]
Thanks for pushing forward on this. r+ with that extra comment.

**Comment by rlr** [comment]
fixed ^ Waiting on travis.

**Comment by rlr** [comment]
yay! merging!

**Pull request merged**